manifest_prod.yml

```diff
@@ -12,4 +14,5 @@ env:
       "/data": "https://fec-prod-web.app.cloud.gov",
       "/regulations": "https://fec-prod-eregs.app.cloud.gov"
     }
-  S3_BUCKET_URL: "https://cg-47928592-406c-4536-8234-99b896e8d57d.s3-us-gov-west-1.amazonaws.com"
+  S3_BUCKET_URL: "https://cg-519a459a-0ea3-42c2-b7bc-fa1143481f74.s3-us-gov-west-1.amazonaws.com"
```

Comment:
Ack!  This was an incorrect change, this S3 bucket is referring to the content bucket!  It needs to change back, it was correct.  This is not related to downloads.